site/documentation/content/api/tenant/index.md

```diff
@@ -279,18 +270,18 @@ The table below contains the properties which are used to configure a tenant's d
 | :------------------------| :-------: | :------------ | :------------ | :---------- |
 | *effective-since*        | *yes*     | *string*      | `-`           | The point in time at which the current settings became effective, i.e. the start of the first accounting period based on these settings. The value MUST be an [ISO 8601 compliant *combined date and time representation in extended format*](https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).|
 | *max-bytes*              | *no*      | *number*      | `-1`          | The maximum number of bytes allowed for the tenant for each accounting period. MUST be an integer. A negative value indicates that no limit is set. |
-| *period*                 | *no*      | *object*      | `-`          | The mode and length of an accounting period, i.e. the data usage can limited based on the defined number of days or on a monthly basis. |
+| *period*                 | *yes*      | *object*     | `-`           | The mode and length of an accounting period, i.e. the data usage can limited based on the defined number of days or on a monthly basis. For more information, please refer to the [resource limits period]({{< relref "#resource-limits-period-configuration-format" >}}).|
```

Comment:
This actually constitutes a breaking change of the (public) Tenant API. I don't think that we can do this in 1.1.0. However, IIRC the Prometheus based resource limits check currently expects to always find this information, doesn't it? If so, we will need to change this and fall back to a default value (e.g. `monthly`) if no resource limits period is specified.
The HTTP management API also does not require this to be specified when defining Data Volume limits for a tenant.

Reply:
I would update the documentation here to set is as _optional_ and a separate PR for setting the default value to _monthly_. WDYT?

Reply:
If we change this to *optional* then the Prometheus based implementation breaks. So, I'd rather have that case handled as part of this PR.

Reply:
As part of this PR, I would change the _period_ as _optional_ and set the default value to _monthly_ for both _DataVolume_ and the _ConnectionDuration_. Just to make sure before making any changes.

Reply:
This PR should not change the period from being optional (as it had been before).
We should probably document, that if the period is not set explicitly, *monthly* is assumed as the default value. The Prometheus based implementation needs to be adapted accordingly.

Reply:
> This actually constitutes a breaking change of the (public) Tenant API. I don't think that we can do this in 1.1.0.

@sophokles73 Please correct me if I am wrong. Removing `DataVolumePeriod` type from Tenant API FMPOV is also a breaking change of the (public) Tenant API in 1.1.0. May be we should remove that  in 2.0.0. Unless the users replace it with the new `ResourceLimitsPeriod` type the build fails.

Reply:
Not sure what you mean by *the build fails*. Which build is that?